lalrpop-test/src/expr_module_attributes.lalrpop

```diff
@@ -1,4 +1,4 @@
-#![cfg_attr(feature = "cargo-clippy", allow(clippy))]
+//#![cfg_attr(feature = "cargo-clippy", allow(clippy))]
```

Comment:
Remove this perhaps? Or move it to an outer attribute instead as the rustc bug/limitation may take a while to be fixed.

Reply:
Removed in 576a30515e7365aa8679161b7070609f65f9109a.